aws-rds-dbcluster/aws-rds-dbcluster.json

```diff
@@ -104,25 +102,11 @@
     },
     "Engine": {
       "description": "The name of the database engine to be used for this DB cluster. Valid Values: aurora (for MySQL 5.6-compatible Aurora), aurora-mysql (for MySQL 5.7-compatible Aurora), and aurora-postgresql",
-      "type": "string",
-      "enum": [
-        "aurora",
-        "aurora-mysql",
-        "aurora-postgresql",
-        "mysql",
-        "postgres"
-      ]
+      "type": "string"
     },
     "EngineMode": {
       "description": "The DB engine mode of the DB cluster, either provisioned, serverless, parallelquery, global, or multimaster.",
-      "type": "string",
-      "enum": [
-        "provisioned",
-        "serverless",
-        "parallelquery",
-        "global",
-        "multimaster"
-      ]
```

Comment:
I am only having two concerns
- This might effect public documentation as it is automatically generated from schema. So values could be removed from there. (We might reach out CFN team)
- Can effect CFN Stack designer tool users as it uses schema to provide autocomplete, validation,....

Reply:
Enums were never published before in the earlier versions of the schema. The documentation is quite static and human-written at the moment. We can add it later on if required.